src/session.rs

```diff
@@ -521,6 +565,24 @@ mod tests {
         assert_eq!(output_tensor[1], 6.0);
     }
 
+    #[test]
+    fn test_run_metadata() {
+        let (session, x_operation, y_operation) = create_session();
+        let mut x = <Tensor<f32>>::new(&[2]);
+        x[0] = 2.0;
+        x[1] = 3.0;
+        let mut step = SessionRunArgs::new();
+        step.add_feed(&x_operation, 0, &x);
+        let output_token = step.request_fetch(&y_operation, 0);
+        // read serialized RunOptions protobuf with trace level set to full.
+        let f = fs::read("test_resources/session/run_options/run_options.bin").unwrap();
```

Comment:
I don't think a separate file is necessary for this, especially since it looks like an empty protobuf anyway.  You could just hard-code the array here.

Reply:
Proto is now hardcoded.